packages/core/src/Stores/notification-store.js

```diff
@@ -922,6 +930,24 @@ export default class NotificationStore extends BaseStore {
                 message: <Localize i18n_default_text='Check your connection.' />,
                 type: 'danger',
             },
+            close_uk_account: {
+                key: 'close_uk_account',
+                header: localize('Your account is scheduled to be closed'),
+                message: localize('Please close all your positions before 25 Apr and withdraw your funds.'),
+                action: {
+                    text: localize('Learn more'),
+                    onClick: () => {
+                        ui.showCloseUKAccountPopup(true);
+
+                        this.removeNotificationByKey({ key: this.client_notifications.close_uk_account.key });
+                        this.removeNotificationMessage({
+                            key: this.client_notifications.close_uk_account.key,
+                            should_show_again: true,
```

Comment:
Is this part of the requirements? seems redundant to display the notification over and over again.